lib/game.rb

```diff
@@ -142,26 +133,44 @@ def play_game
     computer_placement
     player_placement
     @computer.acquire_enemy(@player)
+    @player.acquire_enemy(@computer)
     until @game_over
       render_playspace
       take_turn
     end
     restart_game
   end
 
+  def adjust_board_size
+    letters = ("A".."Z").to_a
+    system "clear"
+    puts "Please enter a number lower than 26"
+    number = gets.chomp.to_i
+
+    until number < 26
+      number = gets.chomp.to_i
+    end
+
+    board_range = letters.slice(number - 1) + number.to_s
```

Comment:
### What does this do?
 - Grabs the largest letter that matches the number input and formats it to the form `G7`